### PR TITLE
Epmcuxda 6583  create i18n provider

### PR DESCRIPTION
**Pull request by AlexeyKaryka**
Add i18n provider component, ru and be locales, initial translates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
     "lint:fix": "eslint ./src/**/*.js App.js --fix"
   },
   "dependencies": {
+    "EventEmitter": "^1.0.0",
```
Comment (AfanasyShevtsov) [inline package.json]: Use fix version here.

File: src/i18n/locales/ru.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "bandingCenter": "Центр Кольцевания Птиц при Акажемии наук Республики Беларусь",
```
Comment (AfanasyShevtsov) [inline src/i18n/locales/ru.json]: Can we have more than 1 level? As an example `registration.password` instead of `password`?

Comment (AlexeyKaryka) [reply src/i18n/locales/ru.json]: Yep, good idea. I propose initially give screen-specific domens to all of the translations, further if there will be doublicates, they can be put on the top level, out of the domen, are you ok with that?

Comment (AfanasyShevtsov) [reply src/i18n/locales/ru.json]: I am okay with duplicates.

File: package.json (hunk 1 above)
Comment (AlexeyKaryka) [reply package.json]: It would appear that 1.0.0 is final version
https://www.npmjs.com/package/EventEmitter

Comment (AlexeyKaryka) [reply package.json]: Or probably you mean to substitute ^ to ~?

**Pull request merged**